curp/src/server/raw_curp/mod.rs

```diff
@@ -792,6 +801,24 @@ impl<C: 'static + Command, RC: RoleChange + 'static> RawCurp<C, RC> {
         self.ctx.leader_tx.subscribe()
     }
 
+    /// Check on the leader that the remove node conf change entry has been synced to the removed follower
+    pub(super) fn can_remove_follower_after_hb(&self, follower_id: ServerId) -> bool {
+        if self.ctx.connects.contains_key(&follower_id) {
+            return false;
+        }
+        let match_index = self.lst.get_match_index(follower_id);
+        let last_conf_change_idx = self.ctx.last_conf_change_idx.load(Ordering::Acquire);
```

Comment:
It seems we can use `Ordering::Relaxed` here.

Reply:
FYI: https://github.com/xline-kv/Xline/pull/460#discussion_r1361413078